app/main.py

```diff
@@ -1,2 +1,82 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+CAPACITY = 8
```

Comment:
These constants as well as Node class a part of dict internal implementation so it would make sense to package everything inside of Dictionary class